src/plugins_types.c

```diff
@@ -914,18 +931,24 @@ ly_type_store_binary(const struct ly_ctx *ctx, const struct lysc_type *type, con
         const struct lysc_node *UNUSED(ctx_node), struct lyd_value *storage, struct lys_glob_unres *UNUSED(unres),
         struct ly_err_item **err)
 {
+    LY_ERR ret = LY_SUCCESS;
     size_t start = 0, stop = 0, count = 0, u, termination = 0;
     struct lysc_type_bin *type_bin = (struct lysc_type_bin *)type;
     char *errmsg;
     int rc = 0;
 
-    LY_CHECK_ARG_RET(ctx, value, LY_EINVAL);
+    if (value == NULL) {
```

Comment:
This change doesn't make sense - if the `value` is NULL, you are goto to free it.